### PR TITLE
Add plan maintainers section

### PR DESCRIPTION
**Pull request by elliott-davis**
Signed-off-by: Elliott Davis <edavis@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @fnichol and @nathenharvey to be potential reviewers

File: MAINTAINERS.md [hunk 1]
```diff
@@ -70,6 +70,21 @@ The web application for the Habitat builder and depot.
...
 
+### Habitat Plans
+
+#### Lieutenant
+
+* [Adam Jacob](https://github.com/adamhjk)
+
+#### Maintainers
+
+* [Fletcher Nichol](https://github.com/fnichol)
```
Comment (smith) [inline MAINTAINERS.md]: Alphabetize it!

**Comment by smith** [comment]
Close enough! 💃 

gif-keyboard-10801314910335782775
ding gif-keyboard-1410973681404248427.gif…]()

**Comment by smith** [comment]
@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit 91dfabb has been approved by `smith`

**Comment by thesentinels** [comment]
:hourglass: Testing commit 91dfabb with merge 91dfabb...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/habitat-sh/habitat/builds/141363357)

**Pull request merged**